requirements-base.txt

```diff
@@ -1,17 +1,17 @@
 django~=3.2
-django-cors-headers==4.1.0
-django-environ~=0.11.2
-django-filter==23.5
+django-cors-headers<=4.1.0
+django-environ<=0.11.2
+django-filter<=23.5
```

Comment:
this is almost certainly incorrect. usually we want to define a minimum supported version, and sometimes also a maximum one (in case we're not compatible with a major release)